src/Offcanvas.tsx

```diff
@@ -221,7 +221,7 @@ const Offcanvas: BsPrefixRefForwardingComponent<'div', OffcanvasProps> =
       const handleHide = useEventCallback(onHide);
 
       bsPrefix = useBootstrapPrefix(bsPrefix, 'offcanvas');
-      const modalBsPrefix = useBootstrapPrefix(undefined, 'modal');
+      const modalBsPrefix = useBootstrapPrefix(undefined, 'offcanvas');
```

Comment:
We don't need `modalBsPrefix` anymore because it's essentially the same as the one above it.

Reply:
Okay. Let me make the necessary changes.